garak/data/calibration/calibration-2025-05.json

```diff
@@ -377,6 +452,26 @@
     "sigma": 0.2638583740585717,
     "sw_p": 0.0016876223608560076
   },
+  "phrasing.FutureTense/mitigation.Prefixes": {
+    "mu": 0.32374526515151514,
+    "sigma": 0.20091650437204153,
+    "sw_p": 0.07835467751575158
+  },
+  "phrasing.FutureTense/specialwords.Prefixes": {
+    "mu": 0.05859375,
+    "sigma": 0.0,
+    "sw_p": NaN
```

Comment:
Please validate that this is an acceptable value for this entry when calculating scoring.